static/app/components/group/issueReplayCount.tsx

```diff
@@ -27,18 +27,23 @@ function IssueReplayCount({groupId}: Props) {
   const countDisplay = count > 50 ? '50+' : count;
 
   return (
-    <Tooltip title={t('This issue has %s replays available to view', countDisplay)}>
+    <StyledTooltip title={t('This issue has %s replays available to view', countDisplay)}>
```

Comment:
you might use the `tn(` translate function to handle "1 replay" vs "2 replays"

Reply:
damnit, just saw this as i was clicking.

i'll send in another pr with that

Reply:
Done! https://github.com/getsentry/sentry/pull/41568